Bloom.md

```diff
@@ -0,0 +1,341 @@
+---
+RFC: 4
+Status: Proposed
+---
+
+# ValkeyBloom Module RFC
+
+## Abstract
+
+The proposed feature is ValkeyBloom which is a Rust based Module that brings a native bloom filter data type into Valkey.
+The module should be API-compatible with Redis Ltd.’s ReBloom Module.
```

Comment:
"The module should be API-compatible with **current** version of  Redis Ltd.’s ReBloom" Module?